examples/chip-tool/commands/clusters/WriteAttributeCommand.h

```diff
@@ -244,8 +244,10 @@ class WriteAttribute : public InteractionModelWriter, public ModelCommand, publi
 
 private:
     // This constructor is private as it is not intended to be used from outside the class.
-    WriteAttribute(chip::ClusterId clusterId, chip::AttributeId attributeId, CredentialIssuerCommands * credsIssuerConfig) :
-        InteractionModelWriter(this), ModelCommand("write", credsIssuerConfig), mClusterIds(1, clusterId),
+    WriteAttribute(chip::ClusterId clusterId, chip::AttributeId attributeId, bool isWritable,
```

Comment:
`WriteAttribute(..., isWritable, ...)` is not obvious about usage. I can infer from the signature that `clusterid, attributeid` represent the path to the attribute I need, credsIssuerConfig gives me the cderentials ... but what is `isWritable?`. It seems to control the command name. How about having:

```cpp
enum class WriteCommandType {
   kWrite, // regular, writable attributes
   kForceWrite, // forced writes, send a write command on something expected to fail
}

....
WriteAttribute(chip::ClusterId clusterId, chip::AttributeId attributeId, WriteCommandType type, ...)
```